documentation/src/main/asciidoc/topics/con_server_hinting.adoc

```diff
@@ -1,7 +1,24 @@
 [id='server-hinting_{context}']
 = Server hinting
 
-The following topology hints can be specified:
+Server hinting helps you to achieve high availability of your data in case of hardware and network failure.
+When you configure server hinting, {brandname} distributes your data across the cluster and stores backup copies of your data on as many servers, racks, and data centers as possible.
+
+NOTE: When you use replicated caches, {brandname} keeps a complete copy of all entries on each node. Server hinting distributes data across the topology so that backup copies of data are stored on different nodes in the topology than the primary owners.
+
+{brandname} uses consistent hashing mechanism to distribute data across nodes.
+The algorithm can potentially move a greater number of segments than necessary during cluster rebalancing.
+You can mitigate this by configuring a larger number of segments.
+However, using a large number of segments can lead to uneven distribution of segments.
+
+Server hinting is suitable for deployments with nodes spread at least on two machines.
+When all the nodes are on the same machine, {brandname} allocates copies of entries in the same location.
+Without server hinting configured, {brandname} might store copies in the same location as the original data.
```

Comment:
It seems like this info should come before the performance thing about the number of segments.